website/screens/utilities/halstack-provider/HalstackProviderPage.tsx

```diff
@@ -0,0 +1,191 @@
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import Example from "@/common/example/Example";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import opinionatedTheme from "./examples/opinionatedTheme";
+import advancedTheme from "./examples/advancedTheme";
+import customTranslations from "./examples/customTranslations";
+import {
+  DxcFlex,
+  DxcParagraph,
+  DxcTable,
+  DxcLink,
+  DxcAlert,
+  DxcTypography,
+  DxcBleed,
+  DxcHeading,
+} from "@dxc-technology/halstack-react";
+import Link from "next/link";
+import PageHeading from "@/common/PageHeading";
```

Comment:
Add a space between the import and the object of the sections please it looks a bit messy.

Reply:
Done.